app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -119,6 +121,25 @@ open class DataStore(
         setupAutoLock()
     }
 
+    private fun deleteCredentials(itemId: String?) {
+        try {
+            if (itemId != null) {
+                backend.delete(itemId)
```

Comment:
just after the `delete` should be an explicit call to `sync`, as with iOS.  This is because the `LoginsStorage` does not automatically trigger a sync, the calling application needs to be explicit.

It's working out for now because opening the item list always seems to trigger a sync.